packages/scene-composer/stories/components/hooks/useSceneMetadataModule.ts

```diff
@@ -0,0 +1,32 @@
+import { TwinMakerSceneMetadataModule, initialize } from '@iot-app-kit/source-iottwinmaker';
+import { useMemo } from 'react';
+
+const region = 'us-east-1';
+// const rociEndpoint = 'https://iottwinmaker.us-east-1.amazonaws.com';
+const rociEndpoint = 'https://gamma.us-east-1.twinmaker.iot.aws.dev';
+
+const useSceneMetadataModule = (source, scene, awsCredentials, workspaceId, sceneId) => {
```

Comment:
prefer named variables instead of large number of parameters that have 'no names', i.e. { source, scene, awsCredentials,,, }

Reply:
Updated to use named variables in parameter